tests/test_interface.sh

```diff
@@ -80,5 +80,89 @@ if output=$(TESTING_SKIP_ROOT_CHECK=1 "$CC_SNAPSHOT" -d mytest 2>&1); then
 else
   fail "Dry-run exited with error: $output"
 fi
+# Srart of custom source path tests
+
+export CC_SNAPSHOT_TAR_PATH="${PWD}/snapshot.tar"
+DEFAULT_TAR="${PWD}/snapshot.tar"
+TESTDIR="/tmp/cc_test"
+
+rm -f "$DEFAULT_TAR"
+rm -rf "$TESTDIR"
```

Comment:
maybe add a check to make sure it's not "/" or assert that it starts with "/tmp/cc_test" before rm rf-ing it.